srcpkgs/procs/template

```diff
@@ -10,13 +10,13 @@ license="MIT"
 homepage="https://github.com/dalance/procs"
 changelog="https://raw.githubusercontent.com/dalance/procs/master/CHANGELOG.md"
 distfiles="https://github.com/dalance/procs/archive/refs/tags/v${version}.tar.gz"
-checksum=9b9b59b79049cf6ae2c39d9cc5b0c5af81411ba898a414fda41f68921c3c9539
+checksum=fa5af0951dc8aa63c0590f8c5c1136594866057704cfb1cdfc22ac3cc49437c6
 
 post_build() {
 	PROCS="target/${RUST_TARGET}/release/procs"
```

Comment:
The general style has become to use post_install and `${DESTDIR}/usr/bin/<name>` instead